addons/website_version/models/experiment.py

```diff
@@ -0,0 +1,300 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+import logging
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from openerp.exceptions import Warning
+from openerp import models, fields, exceptions, SUPERUSER_ID, api
+from openerp.tools.translate import _
+import simplejson
+
+_logger = logging.getLogger(__name__)
+
+
+class Experiment_version(models.Model):
+
+    _name = "website_version.experiment_version"
+
+    version_id = fields.Many2one('website_version.version', string="version_id", required=True, ondelete='cascade')
```

Comment:
The string is what appears in a form view, so better use an english term that will be translated. "version_id" is not a fancy label; the string should be "Version", "Experiment Version", ... if you have things to add, you may use the help, by the way.

Valid for experiment_id, and other occurrences if any.